core/common/src/test/java/alluxio/worker/block/io/LocalFileBlockWriterTest.java

```diff
@@ -24,69 +26,69 @@
 import java.io.File;
 import java.io.IOException;
 import java.nio.ByteBuffer;
-import java.nio.channels.WritableByteChannel;
+import java.nio.file.Files;
+import java.nio.file.Paths;
 
 /**
  * Tests for the {@link LocalFileBlockWriter} class.
  */
 public final class LocalFileBlockWriterTest {
-  private static final long TEST_BLOCK_SIZE = 1024;
+  private static final int TEST_BLOCK_SIZE = 1024;
 
   private LocalFileBlockWriter mWriter;
   private String mTestFilePath;
 
-  /** Rule to create a new temporary folder during each test. */
   @Rule
   public TemporaryFolder mFolder = new TemporaryFolder();
 
-  /** The exception expected to be thrown. */
   @Rule
   public ExpectedException mThrown = ExpectedException.none();
 
-  /**
-   * Sets up the file path and writer before a test runs.
-   */
   @Before
   public void before() throws Exception {
     mTestFilePath = mFolder.newFile().getAbsolutePath();
     mWriter = new LocalFileBlockWriter(mTestFilePath);
   }
 
-  /**
-   * Test for the {@link LocalFileBlockWriter#getChannel()} method.
-   */
-  @Test
-  public void getChannel() throws Exception {
-    WritableByteChannel channel = mWriter.getChannel();
-    Assert.assertNotNull(channel);
+  @After
+  public void after() throws Exception {
+    mWriter.close();
+  }
 
-    ByteBuffer buffer = BufferUtils.getIncreasingByteBuffer((int) TEST_BLOCK_SIZE);
-    Assert.assertEquals(TEST_BLOCK_SIZE, channel.write(buffer));
-    channel.close();
-    Assert.assertEquals(TEST_BLOCK_SIZE, new File(mTestFilePath).length());
+  @Test
+  public void transferFrom() throws Exception {
```

Comment:
Should this test be renamed?

Reply:
will name this accordingly based on the discussion of the method name